src/build.py

```diff
@@ -1603,6 +1606,61 @@ def TestEmtestAsm2Wasm():
       os.path.join(work_dirs.GetTest(), 'emtest-asm2wasm-out'))
 
 
+def TestLLVMTestSuite():
+  outdir = GetBuildDir('llvmtest-out')
+  # The compiler changes on every run, so incremental builds don't make sense.
+  Remove(outdir)
+  Mkdir(outdir)
+  # The C++ tests explicitly link libstdc++ for some reason, but we use libc++
+  # and it's unnecessary to link it anyway. So create an empty libstdc++.a
+  proc.check_call([GetInstallDir('bin', 'llvm-ar'), 'rc', 'libstdc++.a'],
+                  cwd=outdir)
```

Comment:
Hmm.. this seems strange/bad.  I wonder how this runs in other libc++ platforms?

Reply:
Not sure. Probably there are already other compatibility libs on the system. or maybe they really want to link against libstdc++ specifically; but that would be weird given that they run this on macs. so :man_shrugging: